styles/components/banner.css

```diff
@@ -0,0 +1,78 @@
+.banner {
+  background-color: #000;
+  color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 200px;
+  border-radius: 1rem;
+  position: relative;
+  overflow: hidden;
+}
+.banner-badge {
```

Comment:
UI looks awesome, Vineet. Just a suggestion, You can give a line space between two declarations.